templates/minio/minio.py

```diff
@@ -61,14 +60,17 @@ def _minio_sal(self):
             'master_namespace': self.data.get('master').get('namespace'),
             'master_address': self.data.get('master').get('address'),
             'block_size': self.data['blockSize'],
+            'public_port': self.data['nodePort'],
```

Comment:
same comment as the other pr, I'd rather public_port becomes node_port for sake of consistency